sympy/functions/special/zeta_functions.py

```diff
@@ -582,6 +583,44 @@ def _eval_rewrite_as_zeta(self, s, **kwargs):
         return (1 - 2**(1 - s)) * zeta(s)
 
 
+class riemann_xi(Function):
+    r"""
+    Riemann Xi function.
+
+    Examples
+    ========
+
+    The Riemann Xi function is closely related to the Riemann zeta function.
+    The zeros of Riemann Xi function are precisely the non-trivial zeros
+    of the zeta function.
+
+    >>> from sympy import riemann_xi, zeta
+    >>> from sympy.abc import s
+    >>> riemann_xi(s).rewrite(zeta)
+    pi**(-s/2)*s*(s - 1)*gamma(s/2)*zeta(s)/2
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Riemann_Xi_function
+
+    """
+
+
+    @classmethod
+    def eval(cls, s):
+        from sympy import gamma
+        z = zeta(s)
+        if s == 0 or s == 1:
+            return Rational(1,2)
```

Comment:
Instead of importing `Rational`, would it be better to return `S.Half` instead or `Rational(1, 2)`?